content/10.diagnostics.md

```diff
@@ -29,7 +28,7 @@ In this case the test was designed to detect sarbecoviruses, of which SARS-CoV-2
 Considering that no other sarbecoviruses are currently known to infect humans, a positive test indicates that the patient is infected with SARS-CoV-2.
 However, this test is not able to discriminate the genetics of viruses within the sarbecovirus clade.
 
-##### CRISPR test
+#### 4.1.c CRISPR test
 
 Recently, two American companies, Mammoth Biosciences and Sherlock Biosciences, proposed to use CRISPR technology [@doi:10.1016/j.cell.2014.05.010] to increase the speed of Coronavirus diagnostic testing [@doi:10.1101/2020.03.06.20032334].
```

Comment:
```suggestion
Recently, two American companies, Mammoth Biosciences and Sherlock Biosciences, proposed to use CRISPR technology [@doi:10.1016/j.cell.2014.05.010] to increase the speed of coronavirus diagnostic testing [@doi:10.1101/2020.03.06.20032334].
```

Reply:
Is this specific to SARS-CoV-2?

Reply:
Good catch, @cgreene, I missed this when I critiqued this section. Neither technology is mentioned in the first paper cited.
Mammoth seems specific to the virus: "we demonstrated that the CRISPR-Cas12 based detection can distinguish SARS-CoV-2 with no cross-reactivity for related coronavirus strains"
I am a bit out of my depth trying to figure out what's going on with the Sherlock technology. Sherlock Biosciences seems to produce a technology called SHERLOCK that is being [used by the Broad Institute to try to detect SARS-CoV-2](https://www.broadinstitute.org/news/enabling-coronavirus-detection-using-crispr-cas13-open-access-sherlock-research-protocols-and). They are enthusiastic about this application. But [most of the development seems to be coming out of the Broad](https://www.biocentury.com/article/304556/covid-19-could-give-crispr-diagnostics-their-first-proof-of-principle). 
How about:
```suggestion
CRISPR technology has been identified as an approach that could increase the rate of diagnostic testing for COVID-19 [@doi:10.1016/j.cell.2014.05.010].
CRISPR technologies produced by two American companies, Mammoth Biosciences and Sherlock Biosciences, are being used in the development of potential diagnostic tests for SARS-CoV-2 [@doi:10.1101/2020.03.06.20032334; @doi:10.1101/2020.02.26.967026].
```
I will try to recruit someone who knows about CRISPR to review this section.